pkg/cmd/admin/policy/psp_review.go

```diff
@@ -0,0 +1,302 @@
+package policy
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/meta"
+	"k8s.io/kubernetes/pkg/apis/apps"
+	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/templates"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	securityapi "github.com/openshift/origin/pkg/security/api"
+)
+
+// PodSecurityPolicySubjectReview and PodSecurityPolicySelfSubjectReview
+var (
+	subjectReviewLong = templates.LongDesc(`Check whether a User, ServiceAccount, Group can create a Pod.
+	If "User" is specified but not "Group", the it is interpreted as "What if User were not member of any groups".
```

Comment:
~~the~~it is interpreted as "What if User is not a member of any group"

and I wouldn't quote User nor Group.